j2k/testData/fileOrElement/strings/formattedString.kt

```diff
@@ -4,7 +4,7 @@ object A {
             "text3"
 
 
-    val TEXT2 = "text1\n"
-    + "text2\n"
-    + "text3"
+    val TEXT2 = ("text1\n"
```

Comment:
You write comment to proper file? I dont see any problem here. Maybe you mean kt-17379.kt?